rosa_planning/rosa-cloud-expert-prereq-checklist.adoc

```diff
@@ -157,6 +157,16 @@ Prerequisites needed from a networking standpoint.
 
 * Configure your firewall to allow access to the domains and ports listed in xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs[AWS firewall prerequisites].
 
+=== Additional custom security groups
+
+When you create a cluster using an existing VPC, you have the option to add additional custom security groups during cluster creation. Complete these prerequisites before you create the cluster:
+
+* Create the custom security groups in AWS before you create the cluster.
+* Associate the custom security groups with the VPC that you are using to create the cluster. Do not associate the custom security groups with any other VPC.
+* If you are adding more than 2 additional custom security groups, request additional VPC quota for Security groups per network interface.
```

Comment:
We allow up to 5 Security Group IDs.
I don't understand the 2 addtl custom SGs mentioned here. Is that 5 - 3 (openshift reserved)? That doesn't sound like a right way we want to document. do you agree? Because, customer could have 5 or 10 in their quota and we could stay away from repeating AWS documentation here.

Reply:
This was based on a ROSA cluster creating 3 SGs by default (worker, infra, and master), but Guilherme has informed that we only use quota by attaching the worker SG to network interface, so there are 4 SGs that can be added and associated with worker network interface without going over quota on a Classic cluster.

Also, as you mentioned in the GDoc, on HCP clusters this will be lower.

Should we just tell users to confirm their quota for security groups per network interface as a prerequisite when they want to make this change?

Reply:
I've just changed this to the following:

```
* You may need to request additional AWS quota for `Security groups per network interface`.

See the requirements for link:https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/prepare_your_environment/rosa-sts-aws-prereqs#rosa-security-groups_rosa-sts-aws-prereqs[Security groups] for more details.
```